lib/dialects/mssql/query.js

```diff
@@ -63,6 +68,12 @@ Query.prototype.run = function(sql, parameters) {
         var results = [];
 
         var request = new self.connection.lib.Request(self.sql, function(err) {
+
+          if(self.sequelize.options.benchmark){
+            var queryTime = microtime.now() - queryBegin;
+            self.sequelize.log('Executed (' + (self.connection.uuid || 'default') + '): ' + self.sql + ' in ' + queryTime + ' μs', self.options);
```

Comment:
No space between time and `us` is common i believe.
No reason to save a queryTime variable is there?

Please add a space after `if` before `(`